addon-test-support/ember-mocha/index.js

```diff
@@ -27,3 +40,59 @@ export {
   it,
   setResolver
 };
+
+export function setupUnitTest(options) {
```

Comment:
can we move these implementations to dedicated files?

Reply:
Sure